From: Tobias Hansen <thansen@debian.org>
Date: Sun, 21 Dec 2025 16:35:14 +0000
Subject: Declare that sjlj.c requires X/Open 7

Things from the X/Open 7 standard are used. This has to
be declared when building with -std=c99.
---
 libco/sjlj.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libco/sjlj.c b/libco/sjlj.c
index 5af1472..62031a9 100644
--- a/libco/sjlj.c
+++ b/libco/sjlj.c
@@ -4,6 +4,7 @@
 */
 
 #define LIBCO_C
+#define _XOPEN_SOURCE 700
 #include "libco.h"
 #include "settings.h"
 
